docs/api.md

```diff
@@ -350,6 +350,66 @@ class ResetPasswordPage extends React.Component {
 }
 ```
 
+#### ChangePasswordForm
+
+Renders a change password form. The parameter `spToken` is required in order for the token to be validated.
+
+```html
+<ChangePasswordForm spToken={this.props.location.query.sptoken} />
+```
+
+Customize the form by providing your own markup.
+
+```html
+<ChangePasswordForm>
```

Comment:
Well, the docs clarely state: `The parameter `spToken` is required in order for the token to be validated`. Just copy-pasting things without understanding why they work is just bad practice.